web/ui_utils.js

```diff
@@ -113,11 +113,13 @@ function getOutputScale(ctx) {
 
 /**
  * Scrolls specified element into view of its parent.
- * element {Object} The element to be visible.
- * spot {Object} An object with optional top and left properties,
- *               specifying the offset from the top left edge.
+ * @param {Object} element - The element to be visible.
+ * @param {Object} spot - An object with optional top and left properties,
+ *   specifying the offset from the top left edge.
+ * @param {boolean} skipOverflowHiddenElements - Ignore elements that have
```

Comment:
For clarity `Ignore elements` should probably be `Ignore DOM elements instead`.
@timvandermeij If you agree, I'll push an updated patch!